locales/en-US.json

```diff
@@ -161,7 +162,23 @@
   "serverNotFound": "Server not found",
   "serverVersion": "Server version",
   "serverVersionTooLow": "Server version needs to be 10.7.0 or higher",
-  "settings": "Settings",
+  "settings": {
+    "settings": "Settings",
+    "apiKeys": {
+      "apiKeys": "Api Keys",
```

Comment:
```suggestion
      "apiKeys": "API keys",
```

Reply:
This key also duplicates `settingsSections.apiKeys.name`, which we should probably avoid.

Reply:
I have removed the `settingsSection.apiKeys.name`.